components/html-block/html-block.js

```diff
@@ -1,6 +1,16 @@
 import '../colors/colors.js';
 import { css, LitElement } from 'lit-element/lit-element.js';
-import { loadMathJax } from '../../helpers/mathjax.js';
+import { htmlBlockMathRenderer } from '../../helpers/mathjax.js';
+import { requestInstance } from '../../mixins/provider-mixin.js';
+
+let renderers;
+
+const getRenderers = () => {
```

Comment:
Currently, this assumes a single set of renderers provided at the `document` simply as an optimization.  If required, we could request a different set of context-specific renderers for each instance of `d2l-html-block`.